### PR TITLE
Website docs beacon evan

### PR DESCRIPTION
**Pull request by technabors**
This PR moves the beacon documentation from sketch to md format. 

It is a fairly small doc. I am a little worried that the hero-image might be a bit hard to see. Perhaps we can have @etytaylor build a new hero image.

### CODE REVIEW

File: _component_design/help-beacon.md [hunk 1]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
...
 
--   Remove the beacon once it has been viewed.
+### When to use
 
--   Avoid using the beacon for warnings or alerts.
+A beacon is best used in the following scenarios:
 
--   Don't obsure text with a beacon.
+- Show when new products/features have been added to the control panel that improve existing functionality.
+- Show new users the location of commonly used features to aid them in getting started.
```
Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`Show new users the location of commonly used features to aid them in getting started.`/`Showing new users the location of commonly used features to aid them in getting started`

File: _component_design/help-beacon.md [hunk 2]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
...
+- **Do not** use multiple beacons at one time.
```
Comment (nimbinatus) [inline _component_design/help-beacon.md]: Create a new list as these last two bullets don't match the introductory text:
```markdown
There are some cases where beacons should not be used:

- Do **not** use multiple beacons at one time.
- Do **not** use the beacon to alert users of warnings or alerts.
```

File: _component_design/help-beacon.md [hunk 3]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
...
+- **Do not** use the beacon to alert users of warnings or alerts.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Beacons will continually follow this life-cycle giving the impression of a pulse to draw user attention."] [class:"image bg-light border"] %}
```
Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`caption:"Beacons will continually follow this life-cycle giving the impression of a pulse to draw user attention.`/`caption:"Beacons continually follow this lifecycle, giving the impression of a pulse to draw user attention.`

File: _component_design/help-beacon.md [hunk 4]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
...
+![]({{site.url}}/assets/images/components/help/beacon/beacon-hero.svg){:width="80%"}
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Best practice
+
+- Place the beacon next to (left or right) navigation items to show location and label.
```
Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`Place the beacon next to (left or right) navigation items`/`Place the beacon to the right or left of navigation items`

Comment (nimbinatus) [inline _component_design/help-beacon.md]: What does `to show location and label` mean?

File: _component_design/help-beacon.md [hunk 5]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
...
+- Place the beacon next to labels or section headers.
+- Use the beacon in conjuction with the directional pop-over.
```
Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`pop-over`/`popover`

File: _component_design/help-beacon.md [hunk 6]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
...
+- **Do not** use the beacon in leiu of UX best practices with regard to affordance.
```
Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`**Do not**`/`Do **not**` (for all following)

Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`leiu`/`lieu`

Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`with regard to`/`about`

Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`UX`/`user experience`

File: _component_design/help-beacon.md [hunk 7]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
...
+- **Do not** use other forms of notification to deliver the context such as automatic tool tips or directional pop-overs
```
Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`the context such as`/`the context, such as`

Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`tool tips`/`tooltips`

Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`pop-overs`/`popovers`

File: _component_design/help-beacon.md [hunk 8]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
...
+- **Do not** obscure the item of text label with the beacon.
```
Comment (nimbinatus) [inline _component_design/help-beacon.md]: What do you mean by `the item of text label`? Do you mean `the labeled item`?

File: _component_design/help-beacon.md [hunk 9]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
...
-    <div class="ds-beacon-con"><span class="ds-beacon-orange ds-tooltip-source temp-beacon" title="<h3 class='ds-tooltip-h3'>New Feature Title</h3><p class='ds-tooltip-p'>A very short explanation of the new feature and something about what it does.</p><div class='ds-btn-group-tooltip'><button class='ds-btn-sm ds-tooltip-btn' onclick='removeTooltips()' type='button'>Learn More</button><button class='ds-btn-sm-sec ds-tooltip-btn' onclick='removeTooltips()' type='button'>Cancel</button></div>"></div>
-  </div>
 </div>
+
+### Default layout
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+This example shows the multiple states for the beacon. The beacon will animate on a loop to catch the user’s eye. Once the beacon is clicked, it dissapears. The pop-over could generate another beacon to provide a tour experience for more advanced workflows or features.
```
Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`will animate`/`animates`

Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`dissapears`/`disappears`

Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`Once the beacon is clicked`/`Once the user clicks the beacon`

Comment (nimbinatus) [inline _component_design/help-beacon.md]: s/`pop-over`/`popover`

Comment (nimbinatus) [inline _component_design/help-beacon.md]: The last sentence here about the tour experience seems out of place. Move it to the next paragraph and condense with the first sentence there.

File: _component_design/help-beacon.md [hunk 10]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
...
+
+You can string beacons together, to create a tour experience. In this instance, only one at a time, and be sure you prioritize most important or impactful feature first. Each pop-over should
```
Comment (nimbinatus) [inline _component_design/help-beacon.md]: Edit lines 62-65 as:
>To create a tour experience for more advanced features, string together beacons. In this scenario, beacons appear one at a time. Ensure you prioritize demonstrating the most important or impactful feature first. Provide an opt-out option on each popover at every step in the tour. Ideally, track the clicks and drop-offs throughout the tour to improve usefulness for the end user.

File: _component_design/help-beacon.md [hunk 11]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
...
+provide an opt out at every step in the tour.
+
+Ideally we should track the clicks and drop offs so we can tweak the tour to be more useful and informative.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"For full CSS specifications, view the developer documentation"] [class:"image bg-light border"] %}
```
Comment (nimbinatus) [inline _component_design/help-beacon.md]: Add a period at the end of the caption as it's a full sentence. Also, I'm not sure if linking works in captions, but I'd try to generate a link to the developer documentation here (if it's ready).

File: _component_design/help-beacon.md [hunk 12]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/help-beacon.md]: updated.

File: _component_design/help-beacon.md [hunk 13]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
...
 
--   Use one beacon at a time.
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
--   Remove the beacon once it has been viewed.
+### When to use
 
--   Avoid using the beacon for warnings or alerts.
+A beacon is best used in the following scenarios:
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/help-beacon.md]: updated.

File: _component_design/help-beacon.md [hunk 14]
```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
...
 
--   Don't obsure text with a beacon.
+- Show when new products/features have been added to the control panel that improve existing functionality.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/help-beacon.md]: updated.

File: _component_design/help-beacon.md (hunk 1 above)
Comment (technabors) [reply _component_design/help-beacon.md]: updated.

File: _component_design/help-beacon.md (hunk 2 above)
Comment (technabors) [reply _component_design/help-beacon.md]: updated.

File: _component_design/help-beacon.md (hunk 3 above)
Comment (technabors) [reply _component_design/help-beacon.md]: updated.

File: _component_design/help-beacon.md (hunk 4 above)
Comment (technabors) [reply _component_design/help-beacon.md]: updated.

File: _component_design/help-beacon.md (hunk 5 above)
Comment (technabors) [reply _component_design/help-beacon.md]: updated. Also link added. `[popover]({{site.baseurl}}/components/popovers.html).`

File: _component_design/help-beacon.md (hunk 4 above)
Comment (technabors) [reply _component_design/help-beacon.md]: No clue. @lfielder Got any insight on this sentence. After some editing it reads. 

"- Place the beacon to the right or left of navigation items to show location and label." 

What does the location and label point mean?

File: _component_design/help-beacon.md (hunk 6 above)
Comment (technabors) [reply _component_design/help-beacon.md]: updated.

Comment (technabors) [reply _component_design/help-beacon.md]: updated.

Comment (technabors) [reply _component_design/help-beacon.md]: updated.

Comment (technabors) [reply _component_design/help-beacon.md]: updated.

File: _component_design/help-beacon.md (hunk 7 above)
Comment (technabors) [reply _component_design/help-beacon.md]: updated.

Comment (technabors) [reply _component_design/help-beacon.md]: updated and link added.

Comment (technabors) [reply _component_design/help-beacon.md]: updated and link added.

File: _component_design/help-beacon.md (hunk 9 above)
Comment (technabors) [reply _component_design/help-beacon.md]: updated.

File: _component_design/help-beacon.md (hunk 8 above)
Comment (technabors) [reply _component_design/help-beacon.md]: yes. updated.

File: _component_design/help-beacon.md (hunk 9 above)
Comment (technabors) [reply _component_design/help-beacon.md]: updated.

Comment (technabors) [reply _component_design/help-beacon.md]: updated.

Comment (technabors) [reply _component_design/help-beacon.md]: updated and linked... hmmm wonder if this is becoming to many links.

File: _component_design/help-beacon.md (hunk 10 above)
Comment (technabors) [reply _component_design/help-beacon.md]: updated.

File: _component_design/help-beacon.md (hunk 11 above)
Comment (technabors) [reply _component_design/help-beacon.md]: updated.

File: _component_design/help-beacon.md (hunk 9 above)
Comment (technabors) [reply _component_design/help-beacon.md]: Now reads

```
This example shows the multiple states for the beacon. The beacon animates on a loop to catch the user’s eye. Once the user clicks the beacon, it disappears. 

To create a tour experience for more advanced features, string together beacons. In this scenario, beacons appear one at a time. Ensure you prioritize demonstrating the most important or impactful feature first. Provide an opt-out option on each [popover]({{site.baseurl}}/components/popovers.html) at every step in the tour. Ideally, track the clicks and drop-offs throughout the tour to improve usefulness for the end user.
```

**Pull request merged**